requirements_docker.txt

```diff
@@ -4,7 +4,7 @@ natsort==6.0.0
 nose==1.3.7
 numpy==1.21.0
 opencv-python==4.4.0.40
-pandas==1.1.5
+pandas==0.24.2
```

Comment:
That's a pretty low version of Pandas... What breaks when using pandas 1.1.5? I upgraded to that version because the older pandas version we were using was breaking things. 
If we can't find a version that works both on the server and for CI I guess we'll have separate versions in different requirement files, but ideally it would be nice to have the same version everywhere (and a newer version).

Reply:
Yes ... that the versions of the used packages should be updated. In that moment I was looking for a quicker fix, to be able to run microDL for the ongoing project and therefore used pandas version 0.24.2. I think it would be useful to have a working requirements.txt file, even though the the packages are outdated, but we should definitely keep updating them as a todo!

This is the error message with the docker image containing pandas version 1.1.5.
![image](https://user-images.githubusercontent.com/48733135/174160621-e4e14dfa-f1a8-4596-88cd-1f4d69fac4e5.png)

Reply:
That is so strange that downgrading pandas solved this since it seems like it is a bug coming from scikit-image... 
And what was scikit-image doing trying to import an underscored function from numpy?

Maybe we'll have to see if the bug persists with upgraded numpy and skimage. But let's prioritize making the main branch work and then we can revisit versions in a future PR.